ups/verify.table

```diff
@@ -1,13 +1,7 @@
-setupRequired(python_future)
-setupRequired(pyyaml)
-setupRequired(requests)
-setupRequired(numpy)
-setupRequired(astropy)
```

Comment:
I finally noticed this part of the pull request. We are still required to declare a dependency on `astropy` in this package because it uses `astropy` and therefore requires the `eups` `astropy` package to be included (which is a stub that checks that astropy is there). Also `pyyaml`, `requests`, and `numpy` should be listed as dependencies. We do not need to list `pytest_flake8` because `sconsUtils` requires it for now although it doesn't hurt to include it.

Finally, this package still uses `future` so absolutely has to depend on `python_future` until all those `past.builtins` etc imports are removed.

Sorry.